source/gui/addonGui.py

```diff
@@ -38,11 +38,11 @@ def promptUserForRestart():
 	if wx.YES == result:
 		core.restart()
 
+
 class ConfirmAddonInstallDialog(wx.Dialog, DpiScalingHelperMixin):
 	def __init__(self, parent, title, message,
-			userConfirmationMessage=None,
-			errorDialog=False,
-			showAddonInfoFunction=None
+			showAddonInfoFunction,
+			errorDialog=False

```

Comment:
I wonder whether this is possible:

Default message boxes play a Windows sound to indicate whether there is a question or an error. In wx, this behavior is controlled with wx.ICON_WARNING and wx.ICON_ERROR, respectively. I wonder whether adding one of these to the style of the dialog will also trigger this behavior or will at least display the appropriate icon.

Reply:
No I don't think so. I currently have code to display the right icon, but if we wish to get the windows error sound, we could do something like: 
https://stackoverflow.com/questions/13836313/how-can-i-use-the-native-error-icon-and-error-sound-in-windows-using-wxpython

Reply:
That would work.

* For an error: `winsound.PlaySound("SystemHand",winsound.SND_ALIAS | winsound.SND_ASYNC)`
* For a question: `winsound.PlaySound("SystemQuestion",winsound.SND_ALIAS | winsound.SND_ASYNC)`